daemon/api.go

```diff
@@ -333,17 +333,27 @@ func sysInfo(c *Command, r *http.Request, user *auth.UserState) Response {
 }
 
 func sandboxFeatures(backends []interfaces.SecurityBackend) map[string][]string {
-	result := make(map[string][]string, len(backends))
+	result := make(map[string][]string, len(backends)+1)
 	for _, backend := range backends {
 		features := backend.SandboxFeatures()
 		if len(features) > 0 {
 			sort.Strings(features)
 			result[string(backend.Name())] = features
 		}
 	}
-	if len(result) == 0 {
-		return nil
+
+	// fake another security backend :-)
```

Comment:
Nitpick: `// Add information about supported confinement types as a fake backend`

Reply:
nothing to see here, move on